README.md

```diff
@@ -1,140 +1,24 @@
-# [![Unit Tests][20]][21] ![Release][31] [![Downloads][70]][71] [![Lines of code][81]](<>) [![Total lines][80]](<>) [![Files][82]](<>) [![License][50]][51] [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/easyscience/easydiffractionlib/master)
+<img src="https://raw.githubusercontent.com/EasyScience/EasyDiffractionApp/master/resources/images/ed_logo.svg" height="65">
 
-[![](http://github-actions.40ants.com/easyscience/easydiffractionlib/matrix.svg)](https://github.com/easyscience/easydiffractionlib/actions)
+EasyDiffraction is a Python library for the calculation of neutron powder diffraction pattern based on a structural model and refinement of its parameters against experimental data.
```

Comment:
`... for calculating neutron powder...` would probably sound better
also
`... and for refining of its parameters...`

Reply:
fixed